swim/clock.go

```diff
@@ -37,6 +38,10 @@ func (c systemClock) Now() time.Time {
 	return time.Now()
 }
 
+func (c systemClock) AfterFunc(d time.Duration, f func()) *time.Timer {
```

Comment:
Should have a godoc comment. Perhaps you can redirect people to `time.AfterFunc` though, to avoid duplicating the description.